cdapython/_get_unnest_clause.py

```diff
@@ -6,7 +6,11 @@
 
 # A.B.C.D.column ->
 # SELECT DISTINCT(_D.column) FROM TABLE, UNNEST(A) AS _A, UNNEST(_A.B) AS _B, UNNEST(_B.C) AS _C, UNNEST(_C.D) AS _D
-def _get_unnest_clause(col_name):
+
+from typing import Union, Tuple, List
+
+
+def _get_unnest_clause(col_name: str) -> Tuple[str, List[str]]:
```

Comment:
How does this handle the top level fields that are lists like subject_associated_project? I though searching in a list required getting the unnest clause?